org.lflang/src/org/lflang/ASTUtils.java

```diff
@@ -1059,6 +1075,11 @@ public static InferredType getInferredType(Port p) {
         return getInferredType(p.getType(), null);
     }
 
+    //FIXME: modif4watchdogs
```

Comment:
According to the grammar, watchdogs don't have a value or type, so there appears to be nothing actionable here. Remove?